ellie/monopoly.py

```diff
@@ -0,0 +1,154 @@
+from random import randint
+
+class Player:
+    def __init__(self, player_name, player_token):
+        self.name = player_name
+        self.token = player_token
+        self.money = 1500
+        self.properties = []
+        self.position = 0
+        self.in_jail = False
+        self.is_turn = False
+    
+    def __repr__(self):
+        return f"This player, {self.name}, is playing as the {self.token}. \n{self.name} has ${self.money}, is currently on {board[self.position]}, and owns these properties: {self.properties}"
+
+    def roll_dice(self):
+        # Roll two dice and return the results
+        die1 = randint(1,6)
+        die2 = randint(1,6)
+        result = die1 + die2
+        print(f"{self.name} rolled a {result} with a {die1} and a {die2}!")
+        return [die1, die2, result]
+    
+    def update_position(self):
+        # Check the dice roll and update position on the board.
+        result = self.roll_dice()
+        while result[2] > 0:
+            self.position += 1
+            result[2] -= 1
+            # Wrap around board
+            if self.position >= 40:
+                self.position = 0
+                self.money += 200
+                print(f"{self.name} passed Go and collects $200.")
+        if result[0] != result[1]:
+            self.is_turn = False
+        return result
+    
+    def take_turn(self):
+        self.is_turn = True
+        turn_count = 0
+        # Rolls dice and repeats if doubles are achieved, unless it's done three times in a row
+        while self.is_turn or turn_count < 3:
+            turn_count += 1
+            self.update_position()
+            if self.is_turn and turn_count < 3:
+                print(f"{self.name} rolled doubles! {self.name} goes again!")
+            elif turn_count >= 3:
+                print(f"{self.name} rolled doubles three times in a row! {self.name} goes to jail!")
+                self.position = 10
+                self.in_jail = True
+
+
+class Property:
+    def __init__(self, property_name, property_cost, property_house_cost, property_rent0, property_rent1, property_rent2, property_rent3, property_rent4, property_rent5, property_mortgage):
```

Comment:
The `property_` prefix for all these arguments is redundant as the class is already named `Property` and it's pretty clear that these are all related to properties.